src/library.js

```diff
@@ -294,7 +294,7 @@ LibraryManager.library = {
 
     // Memory resize rules:
     // 1. Always increase heap size to at least the requested size, rounded up to next page multiple.
-    // 2a. If MEMORY_GROWTH_LINEAR_STEP == -1, excessively resize the heap geometrically: increase the heap size according to 
+    // 2a. If MEMORY_GROWTH_LINEAR_STEP == -1, excessively resize the heap geometrically: increase the heap size according to
```

Comment:
Personally I prefer not to mix semantic changes with whitespace changes (I think it makes PRs a little harder to review). 

(For this reason I configure my editor to trim whitespace on demand rather than on save.. so it lets me decide when I want to do it).

Not a huge deal we can land this PR as is.

Reply:
Noted. I will watch it for next time